pkg/testgridanalysis/testgridconversion/to_raw_data_test.go

```diff
@@ -0,0 +1,137 @@
+		"\"Installing \"Red Hat Integration - 3scale\" operator in test-nbqyx.Installing \"Red Hat Integration - 3scale\" operator in test-nbqyx Installs Red Hat Integration - 3scale operator in test-nbqyx and creates 3scale Backend Schema operand instance\"",
+		"This test name should not be modified",
+	}
+
+	validationStrings := []string{
+		"Operator results.operator conditions  install_operatorname",
+		"Operator results.operator conditions  upgrade_operatorname",
+		"\"Installing \"Red Hat Integration - 3scale\" operator in test-random.Installing \"Red Hat Integration - 3scale\" operator in test-random Installs Red Hat Integration - 3scale operator in test-random and creates 3scale Backend Schema operand instance\"",
+		"This test name should not be modified",
+	}
+
+	result := processJobDetails(buildFakeJobDetails(testNames), 0, 1)
+
+	assert.NotNil(t, result, "Nil response from processJobDetails")
+
+	// check the keys of the map and validate they match our expectations
+	assert.Equal(t, len(result.TestResults), len(testNames), "Unexpected test resulsts size %d", len(result.TestResults))
+
+	for _, s := range validationStrings {
+		assert.NotNil(t, result.TestResults[s], "Expected non nil test result for %s", s)
+	}
+
+}
+
+func TestCleanTestName(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		vercount int
+		output   string
+	}{
+		{
+			name:     "generic 1",
+			input:    "\"Installing \"Red Hat Integration - 3scale\" operator in test-nbqyx.Installing \"Red Hat Integration - 3scale\" operator in test-nbqyx Installs Red Hat Integration - 3scale operator in test-nbqyx and creates 3scale Backend Schema operand instance\"",
+			vercount: 3,
+		},
+		{
+			name:     "generic 2",
+			input:    "\"Installing \"Red Hat Integration - 3scale\" operator in test-nsyin.Installing \"Red Hat Integration - 3scale\" operator in test-nsyin Installs Red Hat Integration - 3scale operator in test-nsyin and creates 3scale Backend Schema operand instance\"",
+			vercount: 3,
+		},
+		{
+			name:     "generic 3",
+			input:    "\"Installing \"Red Hat Integration - 3scale\" operator in test-piiov.Installing \"Red Hat Integration - 3scale\" operator in test-piiov \"after all\" hook for \"Installs Red Hat Integration - 3scale operator in test-piiov and creates 3scale Backend Schema operand i (...)\"",
+			vercount: 3,
+		},
+		{
+			name:     "generic 4",
+			input:    "\"Installing \"Red Hat Integration - 3scale\" operator in test-piiov.Installing \"Red Hat Integration - 3scale\" operator in test-piiov Installs Red Hat Integration - 3scale operator in test-piiov and creates 3scale Backend Schema operand instance\"",
+			vercount: 3,
+		},
+		{
+			name:     "skip 1",
+			input:    "\"Doesn'tStartWith Installing \"Red Hat Integration - 3scale\" operator in test-.Installing \"Red Hat Integration - 3scale\" operator in test- Installs Red Hat Integration - 3scale operator in test- and creates 3scale Backend Schema operand instance\"",
```

Comment:
Looking great, is it worth doing an output comparison here as well to make sure we didn't modify anything?
Does vercount provide value if we're comparing output strings?